additions.c

```diff
@@ -39,11 +70,17 @@ static JSValue js_getContext()
     char *argv[1] = {""};
 
     glutInit(&argc, argv);
+    DEBUG_GL_CHECK_RAW("glutInit", "%d, %p", argc, argv);
```

Comment:
```suggestion
```

This call causes segfault, let's remove it. I think this is a little bit early place to call `glGetError()`.